tests/ops/integration_tests/saas/connector_runner.py

```diff
@@ -0,0 +1,227 @@
+import logging
+import random
+from typing import Any, Dict, List
+
+from fideslib.cryptography import cryptographic_util
+from sqlalchemy.orm import Session
+
+from fides.api.ops.graph.graph import DatasetGraph
+from fides.api.ops.models.connectionconfig import (
+    AccessLevel,
+    ConnectionConfig,
+    ConnectionType,
+)
+from fides.api.ops.models.datasetconfig import DatasetConfig
+from fides.api.ops.models.policy import Policy
+from fides.api.ops.models.privacy_request import PrivacyRequest
+from fides.api.ops.schemas.redis_cache import Identity
+from fides.api.ops.service.connectors import get_connector
+from fides.api.ops.task import graph_task
+from fides.api.ops.task.graph_task import get_cached_data_for_erasures
+from fides.api.ops.util.collection_util import Row
+from fides.api.ops.util.saas_util import (
+    load_config_with_replacement,
+    load_dataset_with_replacement,
+)
+from fides.ctl.core.config import get_config
+
+CONFIG = get_config()
+logger = logging.getLogger(__name__)
+
+
+class ConnectorRunner:
```

Comment:
This is the main addition, base logic to create the connection_config, dataset_config, and the workflows we need to test (connection_test, access, strict_erasure, non_strict_erasure)